magefile.go

```diff
@@ -10,14 +10,8 @@ import (
 
```

Comment:
Can you create a separate ticket for removing mage? Unless there is a small amount of work left, I am favor of tackling it in a follow up PR.

Reply:
It's already removed in the follow-up PR #127